sdk/cosmos/azure-cosmos/test/test_query_async.py

```diff
@@ -61,25 +62,33 @@ async def test_query_change_feed_with_pk(self):
         # Read change feed without passing any options
         query_iterable = created_collection.query_items_change_feed()
         iter_list = [item async for item in query_iterable]
-        self.assertEqual(len(iter_list), 0)
+        assert len(iter_list) == 0
 
         # Read change feed from current should return an empty list
         query_iterable = created_collection.query_items_change_feed(partition_key=partition_key)
         iter_list = [item async for item in query_iterable]
-        self.assertEqual(len(iter_list), 0)
-        self.assertTrue('etag' in created_collection.client_connection.last_response_headers)
-        self.assertNotEqual(created_collection.client_connection.last_response_headers['etag'], '')
+        assert len(iter_list) == 0
+        if 'Etag' in created_collection.client_connection.last_response_headers:
+            assert created_collection.client_connection.last_response_headers['Etag'] != ''
+        elif 'etag' in created_collection.client_connection.last_response_headers:
+            assert created_collection.client_connection.last_response_headers['etag'] != ''
```

Comment:
Why do we sometimes return `Etag` and sometimes `etag`?
Do our sync tests do this?

Reply:
it seems to be dependent on the OS running it for the async tests - only the ubuntu_py39 has this happen for some reason